src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+//
+// http://csharphelper.com/blog/2015/02/draw-lines-with-custom-end-caps-in-c/
+//
+// Text on path
+// http://csharphelper.com/blog/2018/02/draw-text-on-a-circle-in-c/
+// http://csharphelper.com/blog/2016/01/draw-text-on-a-curve-in-c/
+
+
+// This file should be placed in this address
+// https://github.com/ScottPlot/ScottPlot/tree/master/src/ScottPlot/Plottable
+// and named RadialGaugePlot.cs
+namespace ScottPlot.Plottable
+{
+    /// <summary>
+    /// A radial gauge chart is a graphical method of displaying scalar data in the form of 
+    /// a chart made of circular gauges so that each scalar is represented by each gauge.
+    /// 
+    /// Data is managed using a single array where each element is asigned to each gauge.
+    /// Internally this data is stored in a single array and is converted to angular paramters,
+    /// through ComputeAngularData(), which are more suitable for drawing purposes and stored in a 2D array.
+    /// </summary>
+    public class RadialGaugePlot : IPlottable
+    {
+        #region Properties & fields
+        /// <summary>
+        /// Data to be plotted.
+        /// It's copied from of the data passed to either the constructor or the <see cref="Update(double[])"/> method.
+        /// </summary>
+        protected double[] DataRaw;
+
+        /// <summary>
+        /// Angular data (first column: initial angle; second column: swept angle) computed from <see cref="DataRaw"/>.
+        /// </summary>
+        protected double[,] DataAngular;
+
+        /// <summary>
+        /// The maximum value for scaling the gauges.
+        /// This value is associated to <see cref="StartingAngleGauges"/> + <see cref="AngleRange"/>.
+        /// </summary>
+        protected double MaxScale
+        {
+            get => _MaxScale;
+            set
+            {
+                _MaxScale = value;
+                ComputeAngularData();
+            }
+        }
+        private double _MaxScale;
+
+        /// <summary>
+        /// The minimum value for scaling the gauges.
+        /// This value is associated to <see cref="StartingAngleGauges"/> + <see cref="AngleRange"/>.
+        /// </summary>
+        protected double MinScale
+        {
+            get => _MinScale;
+            set
+            {
+                _MinScale = value;
+                ComputeAngularData();
+            }
+        }
+        private double _MinScale = 0;
+
+        /// <summary>
+        /// The maximum angular interval that the gauges will consist of.
+        /// It takes values in the range [0°-360°], default value is 360°. Outside this range, unexpected side-effects might happen.
+        /// </summary>
+        public double AngleRange
+        {
+            set
+            {
+                // Modify value to be in the range [0, 360]
+                _AngleRange = value > 360 ? 360 : (value < 0 ? 0 : value);
```

Comment:
I think you can use `ReduceAngle` for this

Reply:
I'm not really sure. While `ReduceAngle` should transform any angle into the [0-360] range, here we are doing a cut-off (something like a hard limiter filter in audio processing, or a king of 'angular trimming').

For instante, let's take a value of -365:
- `ReduceAngle` should return 355
- Here we want to get 0

Reply:
> here we are doing a cut-off

This may have been overlooked because double ternary operators are pretty easy to misunderstand 😅 

```cs
_AngleRange = value > 360 ? 360 : (value < 0 ? 0 : value);
```

could be replaced by this which does the same thing but is less likely to be misunderstood

```cs
_AngleRange = value;
_AngleRange = Math.Min(360, _AngleRange);
_AngleRange = Math.Max(0, _AngleRange);
```

I'll probably simplify this further by wrapping that in a function `double TrimAngle(double angle)`

Reply:
My bad, I misunderstood what the purpose of this section was.